press/api/account.py

```diff
@@ -83,8 +85,14 @@ def verify_otp(account_request: str, otp: str):
 
 
 @frappe.whitelist(allow_guest=True)
+@rate_limit(limit=5, seconds=60)
 def resend_otp(account_request: str):
 	account_request: "AccountRequest" = frappe.get_doc("Account Request", account_request)
+
+	# if last OTP was sent less than 30 seconds ago, throw an error
+	if account_request.modified and (frappe.utils.now_datetime() - account_request.modified).seconds < 30:
```

Comment:
checking `modified` value is implicit, maybe store `otp_generated_at` for explicitness?